qa-tests-backend/src/test/groovy/SACTest.groovy

```diff
@@ -62,8 +63,10 @@ class SACTest extends BaseSpecification {
     static final private Integer WAIT_FOR_VIOLATION_TIMEOUT = isRaceBuild() ? 600 : 60
 
     def setupSpec() {
-        // ROX-6260: pre scan the image to avoid missing risk score
-        Services.scanImage(TEST_IMAGE)
+        // ROX-6260: pre scan the image to avoid missing risk score.
+        def img = Services.scanImage(TEST_IMAGE)
```

Comment:
Same test image is define in base specification maybe we can remove definition from this test

https://github.com/stackrox/stackrox/blob/7b98be2e0172026929d2e8e6908125e3c0bdba40/qa-tests-backend/src/test/groovy/BaseSpecification.groovy#L42